MovieApp/MovieApp/ViewController/MenuListController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MenuListController.swift
+//  MovieApp
+//
+//  Created by Julio Gabriel Tobares on 19/04/2022.
+//
+
+import Foundation
+import UIKit
+
+class MenuListController: UITableViewController {
+    
+    //MARK: - Variables
+    var pages = [Constants.SideMenu.movies, Constants.SideMenu.tvShows, Constants.SideMenu.favorites]
+    var darkColor = UIColor(red: 33/255.0, green: 33/255.0, blue: 33/255.0, alpha: 1)
```

Comment:
These are not Variables, they are Constants. And add this UIColor to Constants too.